### ISSUE TITLE
updateTextPos / layoutChildren

### ISSUE THREAD

**Issue by lostspirit1** [jfoenixadmin/JFoenix#1041]
When I add an anchorpane screen with jfoenix textfield to another anchorpane by adding node
i have this error if i do it with textfield i don't have this error

My code:

```
   @FXML
    void atividadeUsuario(ActionEvent event) {
        try {
            Pane atividade = FXMLLoader.load(getClass().getResource("/gerenciamento/usuario/TabAtividade.fxml")); // here have jtextfield 
            conteudo.getChildren().addAll(atividade);
            System.out.println("xdx");
        } catch (IOException ex) {
            Logger.getLogger(HomeController.class.getName()).log(Level.SEVERE, null, ex);
        }
    }

```
```
Exception in thread "JavaFX Application Thread" java.lang.NullPointerException
	at com.jfoenix.skins.JFXTextFieldSkin.updateTextPos(JFXTextFieldSkin.java:109)
	at com.jfoenix.skins.JFXTextFieldSkin.layoutChildren(JFXTextFieldSkin.java:93)
	at javafx.controls/javafx.scene.control.Control.layoutChildren(Control.java:601)
	at javafx.graphics/javafx.scene.Parent.layout(Parent.java:1206)
	at javafx.graphics/javafx.scene.Parent.layout(Parent.java:1213)
	at javafx.graphics/javafx.scene.Parent.layout(Parent.java:1213)
	at javafx.graphics/javafx.scene.Parent.layout(Parent.java:1213)
	at javafx.graphics/javafx.scene.Parent.layout(Parent.java:1213)
	at javafx.graphics/javafx.scene.Parent.layout(Parent.java:1213)
	at javafx.graphics/javafx.scene.Parent.layout(Parent.java:1213)
	at javafx.graphics/javafx.scene.Parent.layout(Parent.java:1213)
	at javafx.graphics/javafx.scene.Parent.layout(Parent.java:1213)
	at javafx.graphics/javafx.scene.Parent.layout(Parent.java:1213)
	at javafx.graphics/javafx.scene.Scene.doLayoutPass(Scene.java:576)
	at javafx.graphics/javafx.scene.Scene$ScenePulseListener.pulse(Scene.java:2482)
	at javafx.graphics/com.sun.javafx.tk.Toolkit.lambda$runPulse$2(Toolkit.java:412)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:389)
	at javafx.graphics/com.sun.javafx.tk.Toolkit.runPulse(Toolkit.java:411)
	at javafx.graphics/com.sun.javafx.tk.Toolkit.firePulse(Toolkit.java:438)
	at javafx.graphics/com.sun.javafx.tk.quantum.QuantumToolkit.pulse(QuantumToolkit.java:562)
	at javafx.graphics/com.sun.javafx.tk.quantum.QuantumToolkit.pulse(QuantumToolkit.java:542)
	at javafx.graphics/com.sun.javafx.tk.quantum.QuantumToolkit.pulseFromQueue(QuantumToolkit.java:535)
	at javafx.graphics/com.sun.javafx.tk.quantum.QuantumToolkit.lambda$runToolkit$11(QuantumToolkit.java:341)
	at javafx.graphics/com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:96)
	at javafx.graphics/com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
	at javafx.graphics/com.sun.glass.ui.win.WinApplication.lambda$runLoop$3(WinApplication.java:174)
	at java.base/java.lang.Thread.run(Thread.java:835)
```

**Comment by lostspirit1** [comment]
i use jdk12

**Comment by jfoenixadmin** [comment]
Hello @lostspirit1,
this is a known JDK12 compatibility issue.

**Comment by jfoenixadmin** [comment]
it's resolved in 2c219040b31d98a1629168a77152178090125ec9, plz use jfoenix 9.0.10
